open-craft/opencraft
# [BB-2388] Host option in `activity_csv` command for Ansible hosts file of active appservers

**Issue**: [BB-2388](https://tasks.opencraft.com/browse/BB-2388)

**Testing Instructions**:
- Try to run the `activity_csv` with `host` option and check that all the `active_appservers` are there or not in the hosts file

**Reviewers**:
